algebird-test/src/test/scala/com/twitter/algebird/CorrelationLaws.scala

```diff
@@ -75,7 +75,9 @@ class CorrelationLaws extends CheckProperties {
   property("the swap method on moments works as you'd think") {
     forAll { l: List[(Double, Double)] =>
       val swapped = CorrelationAggregator(l).swap
-      val reversedInput = CorrelationAggregator.composePrepare[(Double, Double)] { case (x, y) => (y, x) }(l)
+      val fn: ((Double, Double)) => (Double, Double) = { tup => tup.swap }
```

Comment:
just wanted to use `Tuple2.swap` here for which the `Correlation` method is named and show that they are the same. The previous code correctly inlined the implementation of swap, but I thought highlighting the method name equivalence was nice.